AdobeStockClient/Model/SearchParametersProvider/Series.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockClient\Model\SearchParametersProvider;
+
+use AdobeStock\Api\Models\SearchParameters;
+use Magento\AdobeStockClientApi\Api\SearchParameterProviderInterface;
+use Magento\Framework\Api\SearchCriteriaInterface;
+
+/**
+ * Adobe image series filter.
+ */
+class Series implements SearchParameterProviderInterface
```

Comment:
Looks like there is no required custom logic in this parameter provider. Can series be included as part of simple filters?